packages/wdio-reporter/src/stats/suite.ts

```diff
@@ -0,0 +1,47 @@
+import HookStats from './hook'
+import RunnableStats, { Runnable } from './runnable'
+import TestStats from './test'
+
+export interface Suite extends Runnable {
+    type?: string
+    cid?: string
+    title: string
+    fullTitle: string
+    tags?: string[]
+    tests?: TestStats[]
+    hooks?: HookStats[]
+    suites?: SuiteStats[]
+    hooksAndTests?: (HookStats | TestStats)[]
+    description?: string | undefined
```

Comment:
should it be both optional and allow undefined?

Reply:
Fixed